File: app/src/main/java/com/example/android/popularmovies/view/MainView.java
package com.example.android.popularmovies.view;

import com.example.android.popularmovies.model.MovieModel;

import java.util.List;

/**
 * Created by msk-1196 on 6/18/17.
 */

public interface MainView {
    void setMovies(List<MovieModel> movieModels);
}

File: app/src/main/java/com/example/android/popularmovies/ui/view/MainView.kt
package com.example.android.popularmovies.ui.view

import com.example.android.popularmovies.model.MovieModel

/**
 * Created by msk-1196 on 6/18/17.
 */

interface MainView {
    fun setMovies(movieModels: List<MovieModel>)

    fun showLoading()

    fun hideLoading()
}
